db/migrate/20150408162044_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
```

Comment:
Be sure and make sure non nullable fields are set as such as this level as well.  It's not enough to do that in active record only...

Lucas said this in another review... 

Some idiot will come into a production console and type some moronic thing like:
```ruby
User.create(validations: false)
```
and now your data is fucked. Just remember: YOUR DATA IS TRYING TO KILL YOU. Don't let it.